src/library_postprocessing.rs

```diff
@@ -377,10 +377,10 @@ impl Library {
         }
     }
 
-    fn has_subtypes(&self, parent_tid: &TypeId) -> bool {
+    fn has_subtypes(&self, parent_tid: TypeId) -> bool {
         for (tid, _) in self.types() {
             if let Type::Class(ref class) = *self.type_(tid) {
-                if class.parent.as_ref() == Some(parent_tid) {
+                if class.parent == Some(parent_tid) {
```

Comment:
I forgot to add a TODO comment here. I think we want to move the class_hierarchy generation into postprocessing at some point, and then directly make use of the result of it here. Otherwise we have a `O(n²)` loop here and in the class hierarchy code.

Reply:
Added FIXME

Reply:
Thanks :) All good then

Reply:
Sourceview postprocessing take only 10msec from 1.6s total 😉

Reply:
Oh wow. That's **A LOT**!